docs/standard/serialization/system-text-json-migrate-from-newtonsoft-how-to.md

```diff
@@ -0,0 +1,438 @@
+
+### Types without built-in support
+
+`System.Text.Json` doesn't provide built-in support for the following types:
+
+* <xref:System.Data.DataTable> and related types
+* F# types, such as [discriminated unions](../../fsharp/language-reference/discriminated-unions.md), [record types](../../fsharp/language-reference/records.md), and [anonymous record types](../../fsharp/language-reference/anonymous-records.md).
+* Collection types in the <xref:System.Collections.Specialized> namespace
+* <xref:System.Dynamic.ExpandoObject>
+* <xref:System.TimeZoneInfo>
+* <xref:System.Numerics.BigInteger>
+* <xref:System.TimeSpan>
+* <xref:System.DBNull>
+
+### Fields
+
+`Newtonsoft.Json` can serialize and deserialize fields as well as properties. `System.Text.Json` only works with properties.
+
+### Private setters
+
+`Newtonsoft.Json` can use private property setters. `System.Text.Json` supports only public setters.
+
+### Preserve object references and handle loops
+
+By default, `Newtonsoft.Json` serializes by value. For example, if an object contains two properties that contain a reference to the same `Person` object, the values of that `Person` object's properties are duplicated in the JSON.
+
+`Newtonsoft.Json` has a `PreserveReferencesHandling` setting on `JsonSerializerSettings` that lets you serialize by reference:
+
+* A token is added to the JSON created for the first `Person` object.
+* The JSON that is created for the second `Person` object contains that token instead of property values.
+
+`Newtonsoft.Json` also has a `ReferenceLoopHandling` setting that lets you ignore circular references rather than throw an exception.
+
+`System.Text.Json` supports only serialization by value.
+
+### System.Runtime.Serialization attributes
+
+`System.Text.Json` doesn't support attributes from the `System.Runtime.Serialization` namespace, such as `DataMemberAttribute` and `IgnoreDataMemberAttribute`.
+
+### Octal numbers
+
+`Newtonsoft.Json` treats numbers with a leading zero as octal numbers. `System.Text.Json` doesn't allow leading zeroes because the [RFC 8259](https://tools.ietf.org/html/rfc8259) specification doesn't allow them.
+
+### Type name handling
+
+`Newtonsoft.Json` has a `TypeNameHandling` setting that adds type name metadata to the JSON while serializing, and it uses the metadata while deserializing. `System.Text.Json` lacks this feature.
+
+### Populate existing objects
+
+The `Newtonsoft.Json` `JsonConvert.PopulateObject` method deserializes a JSON document to an existing instance of a class, instead of creating a new instance. `System.Text.Json` always creates a new instance of the target type by using the default parameterless constructor.
+
+### Reuse rather than replace properties
+
+The `Newtonsoft.Json` `ObjetCreationHandling` setting lets you specify that objects in properties should be reused rather than replaced during deserialization. `System.Text.Json` always replaces objects in properties.
```

Comment:
```suggestion
The `Newtonsoft.Json` `ObjectCreationHandling` setting lets you specify that objects in properties should be reused rather than replaced during deserialization. `System.Text.Json` always replaces objects in properties.
```